### PR TITLE
Dispatch negotiated result

### PR DESCRIPTION
**Pull request by jaapromijn**
Dispatch the content negotiated result so it can be altered in events.

### CODE REVIEW

File: src/Content/ResultServiceLocator.php [hunk 1]
```diff
@@ -9,8 +9,10 @@
 
 namespace GisoStallenberg\Bundle\ResponseContentNegotiationBundle\Content;
 
+use GisoStallenberg\Bundle\ResponseContentNegotiationBundle\Event\NegotiatedResultDataEvent;
 use Negotiation\Accept;
 use Negotiation\Negotiator;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
```
Comment (gisostallenberg) [inline src/Content/ResultServiceLocator.php]: I'd rather have the interface from symfony/event-dispatcher-contracts which is a requirement of symfony/event-dispatcher 4.4 and 5.0

**Review by gisostallenberg — CHANGES_REQUESTED** [review]
Nice addition, I only would like the interface to be changed.

File: src/Content/ResultServiceLocator.php (hunk 1 above)
Comment (jaapromijn) [reply src/Content/ResultServiceLocator.php]: Updated the interface 👍

**Comment by jaapromijn** [comment]
Updated the interface

**Review by gisostallenberg — APPROVED** [review]

**Pull request merged**